ocfweb/api/session_tracking.py

```diff
@@ -38,21 +43,18 @@ def log_session(request):
 
     try:
         body = json.loads(request.body.decode('utf-8'))
-
         host = _get_desktops().get(remote_ip)
-        user = body.get('user')
-        state = body.get('state')
-
-        if not user:
-            raise ValueError('Invalid user "{}"'.format(user))
 
-        if host is None:
+        if not host:
             raise ValueError('Invalid host "{}"'.format(host))
 
-        if _session_exists(host, user):
-            _refresh_session(host, user)
-        elif state == 'inactive':
+        state = State[body.get('state')]
+        user = body.get('user')
+
+        if state in (State.inactive, State.cleanup) or not user:
```

Comment:
Given `inactive` and `cleanup` are just treated the same here, I don't see why 3 states are needed.